dotnet-60/linux/Dockerfile

```diff
@@ -1,4 +1,21 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0
+FROM mcr.microsoft.com/dotnet/sdk:6.0-bullseye-slim
```

Comment:
Is this required? Image `6.0` and `6.0-bullseye-slim` should be the same on Linux.

Reply:
Fixed, I will use the shorter tag.

Reply:
Okay. I think it is better, because if later they update the `mcr.microsoft.com/dotnet/sdk:6.0` to a newer Debian version, then we will automatically depend on that.